draft-ietf-ppm-dap.md

```diff
@@ -1491,6 +1492,15 @@ shares in its encrypted state, thereby offloading this state to the leader.)
 This is due to the requirement that the batch interval respect the boundaries
 defined by the DAP parameters. (See {{batch-parameter-validation}}.)
 
+# Compliance Requirements {#compliance}
```

Comment:
This make it sound like we only intend to support one suite. It may be worth adding a note here that we will consider adding new MTI suites as needed.